### PR TITLE
CFP reminder post

### PR DESCRIPTION
**Pull request by plaindocs**
## Before publishing 

- [x] tweak date 

## After publishing

- [ ] post in slack
- [ ] post on twitter
- [ ] post on linked in 

----
📚 Documentation preview 📚: https://writethedocs-www--2068.org.readthedocs.build/

### CODE REVIEW

File: docs/conf/portland/2024/news/cfp-reminder.rst [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+- Conference Party
+
+We're excited to see you all in Portland again soon.
+
+Opportunity Grants
+------------------
+
+Interested in attending but need extra financial support? We strive to provide an accessible conference for folks of all backgrounds to join. Our opportunity grant application is open until    **{{ grants.ends }}**.
+
+View our `Opportunity Grants page <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/opportunity-grants/>`_ for information and the grant application, .
```
Comment (janine-c) [inline docs/conf/portland/2024/news/cfp-reminder.rst]:
```suggestion
View our `Opportunity Grants page <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/opportunity-grants/>`_ for information and the grant application.
```

File: docs/conf/portland/2024/news/cfp-reminder.rst [hunk 2]
```diff
@@ -0,0 +1,84 @@
+:template: {{year}}/generic.html
+
+.. post:: Jan 03, 2024
```
Comment (plaindocs) [inline docs/conf/portland/2024/news/cfp-reminder.rst]:
```suggestion
.. post:: Jan 04, 2024
```

File: docs/conf/portland/2024/news/cfp-reminder.rst [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+Interested in attending but need extra financial support? We strive to provide an accessible conference for folks of all backgrounds to join. Our opportunity grant application is open until    **{{ grants.ends }}**.
+
+View our `Opportunity Grants page <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/opportunity-grants/>`_ for information and the grant application.
+
+
+Support our community by sponsoring
+-----------------------------------
+
+Every year we are fortunate to have the support of like-minded organizations and communities,
+who believe in the collaborative and welcoming community spirit and help us make our events great.
```
Comment (ericholscher) [inline docs/conf/portland/2024/news/cfp-reminder.rst:75]:
```suggestion
who believe in the collaborative and welcoming community spirit and help us make our events great.

Thanks so much to the following companies for supporting the conference this year:

.. datatemplate::
   :source: /_data/{{shortcode}}-{{year}}-config.yaml
   :template: {{year}}/sponsors-simplelist.rst
```

We should be listing sponsors in each email... probably makes sense to add this to the template or something, but baby steps :)

File: docs/conf/portland/2024/news/cfp-reminder.rst [hunk 4]
```diff
@@ -0,0 +1,84 @@
+:template: {{year}}/generic.html
+
+.. post:: Jan 04, 2024
+   :tags: {{shortcode}}-{{year}}, website, cfp, tickets
```
Comment (ericholscher) [inline docs/conf/portland/2024/news/cfp-reminder.rst]:
```suggestion
   :tags: {{shortcode}}-{{year}}, cfp, tickets, grants
```

File: docs/conf/portland/2024/news/cfp-reminder.rst [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+
+Call for Proposals and Grant Application Reminder
+===================================================
+
+Happy New Year, fellow documentarians!
+
+This is your scheduled reminder to kick off 2024 by `submitting a talk <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/cfp/>`__ to Write the Docs Portland! You can do so until **{{ cfp.ends }}**.
+
+In other conference news, `tickets are on sale <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/tickets/>`_, and you can still `apply for an opportunity grant <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/opportunity-grants/>`_, read on for more information on any of that good stuff.
```
Comment (ericholscher) [inline docs/conf/portland/2024/news/cfp-reminder.rst]:
```suggestion
In other conference news, `tickets are on sale <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/tickets/>`_, and you can still `apply for an opportunity grant <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/opportunity-grants/>`_, read on for more information.
```

File: docs/conf/portland/2024/news/cfp-reminder.rst [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+**The Portland conference will be held on {{ date.short }}**, in Revolution Hall.
+
+The `full event website <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/>`_ is live and already includes many details. We'll be updating it with more information as we get closer to the event.
+
+If you're excited to join us again in person,
+go ahead and `register for your ticket now <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/tickets/>`_!
+We are still planning to limit capacity of the event to ensure that we have a safe and community-focused experience for everyone,
+so we expect tickets to sell out a few weeks before the conference.
+
+Similar to our past conferences, we're going to keep as much of the spirit of the event as possible:
```
Comment (ericholscher) [inline docs/conf/portland/2024/news/cfp-reminder.rst]:
```suggestion
Similar to our past conferences, we're going to have a wide range of ways to engage:
```

File: docs/conf/portland/2024/news/cfp-reminder.rst [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+- Sponsor Expo + Job Fair
+- Welcome Reception
+- Conference Party
+
+We're excited to see you all in Portland again soon.
+
+Opportunity Grants
+------------------
+
+Interested in attending but need extra financial support? We strive to provide an accessible conference for folks of all backgrounds to join. Our opportunity grant application is open until    **{{ grants.ends }}**.
```
Comment (ericholscher) [inline docs/conf/portland/2024/news/cfp-reminder.rst]:
```suggestion
Interested in attending but need extra financial support? We strive to provide an accessible conference for folks of all backgrounds to join. Our opportunity grant application is open until **{{ grants.ends }}**.
```

**Review by ericholscher — COMMENTED** [review]
Looks good, gonna merge in my changes then 👍  it

**Review by ericholscher — APPROVED** [review]
🚢

File: docs/conf/portland/2024/news/cfp-reminder.rst (hunk 5 above)
Comment (plaindocs) [reply docs/conf/portland/2024/news/cfp-reminder.rst]: :-D ok ok

**Comment by plaindocs** [comment]
Cheers!

**Pull request merged**